clang/docs/analyzer/checkers.rst

```diff
@@ -1288,6 +1288,34 @@ by explicitly marking the ``size`` parameter as sanitized. See the
     delete[] ptr;
   }
 
+.. _optin-taint-TaintedDiv:
+
+optin.taint.TaintedDiv (C, C++, ObjC)
+"""""""""""""""""""""""""""""""""""""
+This checker warns when the denominator in a division
+operation is a tainted (potentially attacker controlled) value.
+If the attacker can set the denominator to 0, a runtime error can
+be triggered. The checker warns if the analyzer cannot prove
+that the denominator is not 0 and it is a tainted value.
```

Comment:
```suggestion
be triggered. The checker warns when the denominator is a tainted value and the
analyzer cannot prove that it is not 0.
```
Swap these two conditions to clarify that  "is a tainted value" is not within the "the analyzer cannot prove that" block.